app/models/thesaurus_setting.rb

```diff
@@ -1,2 +1,3 @@
 class ThesaurusSetting < ApplicationRecord
```

Comment:
don't you think this should fall We have the Thesaurus namespace for Model. This should fall under the Thesaurus namespace ?

Thesaurus::Setting OR something like this Thesaurus::ThesaurusSetting

Reply:
Changes implemented. Please review.